packages/wix-ui-core/package.json

```
@@ -65,7 +65,7 @@
     "storybook-start": "npm run storybook",
     "storybook:visual": "start-storybook -c ./.storybook/visual -p 6007",
     "start": "run-p yoshi-start storybook-start",
-    "release": "npx @wix/yoshi@4 release && npx teamcity-surge-autorelease@^1.0.0",
+    "release": "npx @wix/yoshi@4 release && npx build-storybook && npx janet deploy --ci",
```

```suggestion
    "release": "npx @wix/yoshi@4 release && npx build-storybook --package @storybook/react@5 && npx janet deploy --ci",
```

executing `npx build-storybook` would try to install `build-storybook`
package and run `build-storybook` from `node_modules/.bin`. But in this
case `build-storybook` is coming from `@storybook/react` package.

with `--package` flag we can set which package to get this binary from.

---

Nice!

It's true, cause I tried it with `npx build-storybook` and it failed with
`npm ERR! 404 Not Found - GET http://npm.dev.wixpress.com/build-storybook - not_found`